source/OptimaJet.DWKit.StarterApplication/package.json

```diff
@@ -12,46 +12,46 @@
   "author": "OptimaJet",
   "license": "SEE LICENSE IN eula.txt",
   "dependencies": {
-    "ajv": "5.5.2",
-    "babel-core": "6.26.3",
-    "babel-eslint": "8.2.6",
-    "babel-loader": "7.1.5",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-es3-member-expression-literals": "6.22.0",
-    "babel-plugin-transform-es3-property-literals": "6.22.0",
-    "babel-plugin-transform-react-jsx": "6.24.1",
-    "babel-plugin-typecheck": "3.9.0",
-    "babel-preset-env": "1.7.0",
-    "babel-preset-react": "6.24.1",
-    "babel-preset-stage-2": "6.24.1",
-    "deepmerge": "1.5.2",
-    "es6-promise": "4.2.5",
-    "hard-source-webpack-plugin": "0.6.12",
-    "history": "4.7.2",
-    "isomorphic-fetch": "2.2.1",
-    "json5": "0.5.1",
-    "react": "16.6.0",
-    "react-data-grid": "4.0.8",
-    "react-data-grid-addons": "4.0.8",
-    "react-dom": "16.6.0",
-    "react-hot-loader": "1.3.1",
-    "react-redux": "5.1.0",
-    "react-router": "4.3.1",
-    "react-router-dom": "4.3.1",
-    "react-router-redux": "5.0.0-alpha.9",
-    "redux": "4.0.1",
-    "redux-actions": "2.6.4",
-    "redux-devtools-extension": "2.13.5",
-    "redux-thunk": "2.3.0",
-    "reflux": "6.4.1",
-    "semantic-ui-react": "0.80.2",
-    "url": "0.11.0",
-    "webpack": "3.12.0",
-    "webpack-dev-server": "2.11.3",
-    "babel-polyfill": "6.26.0",
+    "ajv": "^5.5.2",

```

Comment:
Before, all these dependencies had specific versions that they were supposed to be using.  Now they have the latest version (using the ^).  This is from dwkit 2.6 source (see https://github.com/sillsdev/dwkit-source)